jsk_panda_robot/jsk_panda_accessories/README.md

```diff
@@ -1,3 +1,7 @@
+
 # Tool mounter
 Mounting any tools onto panda-hand.  Plate mounter has M3 screw wholes placed by 10mm x 10mm.
 ![](https://raw.githubusercontent.com/ykawamura96/franka_demos/mech/add_adapter_parts/franka_accessories/panda_wrist/hand_adapter.png)
+# gripper
```

Comment:
@softyanija 
Could you replace the name with specific name so that we can know the functionality of it? e.g.) gripper -> cable picking gripper, tweezers gripper, or whatever you think is good.

Reply:
@softyanija 
Also, please modify the file path `gripper` to your name. 
```
jsk_panda_robot/jsk_panda_accessories/gripper/panda-hand_tweezer_v2_left.SLDPRT
->  jsk_panda_robot/jsk_panda_accessories/**something**/panda-hand_tweezer_v2_left.SLDPRT
```

Reply:
I replace the name "gripper" to "tweezers". and rewrite README.md